src/python/pants/backend/go/subsystems/gotest.py

```diff
@@ -152,10 +165,14 @@ class GoTestSubsystem(Subsystem):
         ),
     )
 
-    def coverage_output_dir(self, distdir: DistDir, import_path: str) -> PurePath:
+    def coverage_output_dir(self, distdir: DistDir, address: Address, import_path: str) -> PurePath:
+        target_spec = address.spec_path.replace(os.sep, ".")
         import_path_escaped = import_path.replace("/", "_")
```

Comment:
Should there be a check here to warn about using `import_path_escaped`?

Reply:
A deprecation warning would make sense, yeah.